dd-sdk-android/src/test/kotlin/com/datadog/android/webview/internal/rum/WebViewRumEventConsumerTest.kt

```diff
@@ -131,6 +133,29 @@ internal class WebViewRumEventConsumerTest {
 
     // region View
 
+    @Test
+    fun `M send a noop WebViewEvent W consume { View event }`(forge: Forge) {
+        // When
```

Comment:
minor: this should be probably `Given` and call itself only behind `When`, same for new tests below

Reply:
Yes...I forgot to change those as in the first place I wanted to go with parameterized tests there but realized in the end that will not work :)